rotkehlchen/tests/api/test_user_assets.py

```diff
@@ -889,7 +895,7 @@ def test_exporting_user_assets_list(rotkehlchen_api_server, globaldb, with_custo
         coingecko='YAB',
         cryptocompare='YAB',
     ))
-    with tempfile.TemporaryDirectory() as path:
+    with tempfile.TemporaryDirectory(ignore_cleanup_errors=True) as path:
```

Comment:
Interesting. I see what this does from the docs (https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory). Can you maybe explain why in a small comment next to this, why this is needed for Windows tests? As the reader of the code will end up wondering in the future.